File: backend/trivia-data/src/main/java/com/delacruz/trivia/model/Question.java
package com.ectrvia.trivia.model;

import lombok.Data;
import java.util.List;

@Data
public class Question {
    private Long id;
    private Long roomId;
    private Long categoryId;
    private String questionText;
    private Integer questionOrder;
    private Integer correctAnswerIndex;
    private Integer timerSeconds;
    private List<Answer> answers;
}

File: ECTriviaApp/app/src/main/java/com/delacruz/ectrivia/data/model/Question.kt
package com.ectrvia.ectrivia.data.model

data class Question(
    val id: Long,
    val questionText: String,
    val answers: List<Answer>,
    val questionOrder: Int,
    val correctAnswerIndex: Int = 0,
    val timerSeconds: Int
)

data class Answer(
    val index: Int,
    val text: String
)
